src/ethereum_test_tools/exceptions/exceptions.py

```diff
@@ -92,6 +92,75 @@ class TransactionException(ExceptionBase):
     If a transaction with any of these exceptions is included in a block, the block is invalid.
     """
 
+    ADDRESS_TOO_SHORT = auto()
+    """
+    Transaction `to` is not allowed to be less than 20 bytes
```

Comment:
Sorry for this nit, but as this will get included in the generated doc, it's nice if we add punctuation to help readability.
```suggestion
    Transaction `to` is not allowed to be less than 20 bytes.
```